docs/api/v3.rst

```diff
@@ -0,0 +1,77 @@
+V3 Specification Implementation(``zarr._storage.v3``)
+=====================================================
+
+This module contains the implementation of the `Zarr V3 Specification <https://zarr-specs.readthedocs.io/en/latest/core/v3.0.html#zarr-core-specification-v3-0>`_.
```

Comment:
This link doesn't work.

Maybe an [intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html) reference could be used here? Like 

```
:external+zarr-specs:ref:`Zarr V3 Specification <specs/v3>`
```

Reply:
Thanks for pointing it out, @ivirshup. This happened because of https://github.com/zarr-developers/zarr-specs/pull/204.
I've fixed the broken links manually for now. 

@joshmoore, please let me know if we want to go with the inter-sphinx extension.